Zilon.Core/Zilon.Core/MapGenerators/CellularAutomatonStyle/CellularAutomatonMapFactory.cs

```diff
@@ -97,9 +91,13 @@ private CreateMatrixResult CreateInner(int targetRegionDraftCount, int chanceToS
                     matrix = new Matrix<bool>(newMap, matrix.Width, matrix.Height);
                 }
 
-                var maxtrixScales = CreateScaledMatrix(matrix);
+                // Растягиваем матрицу на 4, чтобы в единичную ячейку матрицы клеточного автомата
+                // мог помещаться персонаж размером в 7 узлов.
+                // Отключено, потому что сейчас слишком долгая генерация + туман войны на больших плошадях тормозит
+                //const int SCALE_FACTOR = 4;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)